packages/core/src/index.ts

```diff
@@ -1,37 +1,38 @@
 console.warn('REAtom still work in progress, do not use it in production')
 
-type NodeId = string
-
-type ActionType = string
```

Comment:
Please, return ActionType and NodeId. It will helpful with nominal types (https://twitter.com/WrocTypeScript/status/1168498435110121472?s=20)

Reply:
[Here](https://github.com/artalar/reatom/blob/1f740b64932658c257da41a64eb5e28160629bb2/packages/core/src/index.ts#L92) conflict with type cast from `NodeId` to `ActionType`